### PR TITLE
Medical Treatment - Settings to limit IVs to certain locations

### PR DESCRIPTION
**Pull request by pterolatypus**
**When merged this pull request will:**
- Adds two settings to limit use of a) saline and b) blood and plasma to specific locations
- Settings are separate to allow more differentiation of the IV types in the future
- Needs translations

Additionally, I noticed that the Treatment subcategory in settings is getting pretty big. Would it be worth moving this to its own category?

### CODE REVIEW

File: addons/medical_treatment/initSettings.sqf [hunk 1]
```diff
@@ -224,6 +224,24 @@
...
 ] call CBA_settings_fnc_init;
 
+[
+    QGVAR(locationSaline),
+    "LIST",
+    [LSTRING(LocationSaline_DisplayName), LSTRING(LocationSaline_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [[0, 1, 2, 3, 4], [ELSTRING(common,Anywhere), ELSTRING(common,Vehicle), LSTRING(MedicalFacilities), LSTRING(VehiclesAndFacilities), ELSTRING(common,Disabled)], 0],
+    true
+] call CBA_settings_fnc_init;
```
Comment (mharis001) [inline addons/medical_treatment/initSettings.sqf:234]: `CBA_fnc_addSetting`.

**Review by mharis001 — COMMENTED** [review]
Why keep plasma and blood together while splitting saline? I think it would be better if all three are controlled by one setting or each has their own.

**Comment by pterolatypus** [comment]
> Why keep plasma and blood together while splitting saline?

It was because I plan to write a separate PR changing how the different fluids work, but on second thoughts I can just split them out if and when I actually do that.

File: addons/medical_treatment/initSettings.sqf (hunk 1 above)
Comment (pterolatypus) [reply addons/medical_treatment/initSettings.sqf]: changed

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**